### ISSUE TITLE
Reviewer chooser is missed in creating PR form

### ISSUE THREAD

**Issue by wolfogre** [go-gitea/gitea#21085]
### Description

Reviewer chooser is missed in creating PR form, users can choose reviewers only after creating the PR.

### Gitea Version

main branch at `b5d21c0`

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots

<img width="997" alt="image" src="https://user-images.githubusercontent.com/9418365/188584617-b06754f2-5422-4e2a-be28-c50a1e718153.png">

### How are you running Gitea?

Use command-line.

**Comment by jpraet** [comment]
duplicate of #26289 and #13905
